core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -306,7 +332,9 @@ private val newsResource = NewsResource(
     id = "1",
     episodeId = "1",
     title = "Title",
-    content = "Content",
+    content = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor" +
+        " incididunt ut labore et dolore magna aliqua. Habitant morbi tristique senectus et netus" +
+        " et malesuada fames.",
```

Comment:
If you want to, you can replace this preview element with `previewNewsResources[0]`

Reply:
@calren can you address this comment as well? :)

Reply:
yup, done!